### ISSUE TITLE
String not Boolean and path incorrect

### ISSUE THREAD

**Issue by AsaLap** [AuReMe/mpwt#70]
Hi,

The optional parameters "taxon_id" in the _mpwt.multiprocess_pwt_ function is not an _optional boolean_ as said in the README but an _optional str_.

Also, the bash command you suggest to add _Pathway Tools_ to the PATH variable should start with a slash "/" and you may also suggest to source the .bashrc file by doing 
```bash
echo 'export PATH="$PATH:/your/install/directory/pathway-tools:"' >> ~/.bashrc
$ source ~/.bashrc
```
I personnaly encountered some issues with the two problems raised here, so I'm reaching out to you so that it can help someone else.

Thanks for your work though,
Antoine

**Comment by ArnaudBelcour** [comment]
Hi Antoine,

Thank your for your issue.

I am sorry that you encounter these problems. I have fixed them in the branch [mpwt_0.7.0](https://github.com/AuReMe/mpwt/commit/9c3d2789d888d2139a25af3515488cd61bef6860).

And until the version 0.7.0 is released, this issue will stay open so people having the same issues could find the solution.

Best regards,
Arnaud.

**Comment by ArnaudBelcour** [comment]
This should have been fixed with the release of the 0.7.0 (#72).
